README.md

```diff
@@ -0,0 +1,272 @@
+<img src="https://raw.githubusercontent.com/shabados/presenter/dev/resources/icon.png" width="128" alt="Shabad OS">
+
+# Database
+
+A digital representation of Sikh Bani and other Panthic texts with a public logbook of sangat-sourced corrections
+
+[![NPM Version][npm-image]][npm-url]
+[![NPM Downloads][downloads-image]][downloads-url]
+[![Release][release-image]][release-url]
+[![Next Release][next-image]][next-url]
+
+**Table of Contents**
+
+- [Introduction](#introduction)
+  - [Features](#features)
+  - [Screenshots](#screenshots)
+    - [Proofreading Flow [GIF]](#proofreading-flow-gif)
+    - [Database Schema Layout](#database-schema-layout)
+    - [NPM JS package API [GIF]](#npm-js-package-api-gif)
+- [Community](#community)
+- [Contributing](#contributing)
+- [People](#people)
+- [Feedback](#feedback)
+- [Related Projects](#related-projects)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+- [Gurbani and Panthic Compositions](#gurbani-and-panthic-compositions)
+  - [Sri Guru Granth Sahib Ji](#sri-guru-granth-sahib-ji)
+  - [Sri Dasam Granth](#sri-dasam-granth)
+  - [Bhai Gurdaas Ji](#bhai-gurdaas-ji)
+  - [Bhai Nand Laal Ji](#bhai-nand-laal-ji)
+- [Acknowledgments](#acknowledgments)
+
+## Introduction
+
+Shabad OS Database is an open source project with ongoing development, project management, and marketing made possible entirely by the support and dedication of unpaid volunteers.
+
+### Features
+
+- Open-source
+  - Content, history, and modifications are publicly accessible

```

Comment:
```suggestion
  - Content, history, and modifications are publicly accessible & auditable

```

Reply:
To make multiline suggestions, just hold and drag the plus symbol